src/ConfigServiceProvider.php

```diff
@@ -18,14 +18,14 @@
     const SETTING_SEPARATOR = 'separator';
 
     /**
-     * @var array
+     * @var ApplicationConfig
      */
     private $config;
 
     /**
-     * @var AggregateServiceProvider
```

Comment:
@tomphp 

Can we also remove the now unused import for `TomPHP\ConfigServiceProvider\League\AggregateServiceProvider`?